### PR TITLE
Fine tune WC email customizer [MAILPOET-2520]

### PR DESCRIPTION
**Pull request by wxa**

### CODE REVIEW

**Comment by wxa** [comment]
@amine-mp Updated the branch according to the changed specs, the only departure is a slightly different notice when WC is disabled, see my latest comment in the Jira task.

File: lib/AdminPages/Pages/NewsletterEditor.php [hunk 1]
```diff
@@ -54,29 +54,38 @@ function render() {
...
       $woocommerce_template_id
       && $newsletter_id === $woocommerce_template_id
-      && (
-        !$this->woocommerce_helper->isWooCommerceActive()
-        || !(bool)$this->settings->get('woocommerce.use_mailpoet_editor', false)
-      )
+      && !$this->woocommerce_helper->isWooCommerceActive()
     ) {
-      header('Location: admin.php?page=mailpoet-settings&enable-customizer-notice#woocommerce', true, 302);
+      $location = 'admin.php?page=mailpoet-settings&enable-customizer-notice#woocommerce';
```
Comment (amine-mp) [inline lib/AdminPages/Pages/NewsletterEditor.php]: From the specs
> If the user tries to access the template editor URL, redirect him to `?page=mailpoet-newsletters` and ...

But you redirect to settings page here.

Comment (wxa) [reply lib/AdminPages/Pages/NewsletterEditor.php]: I guess it's a typo in specs, we changed them to redirect to the message which is in the Settings (see last 3 comments in the Jira task).

Comment (amine-mp) [reply lib/AdminPages/Pages/NewsletterEditor.php]: I see, that makes sense.

**Pull request merged**